sig-instrumentation/annual-report-2024.md

```diff
@@ -30,55 +43,54 @@
-    - [2305 - Dynamic Cardinality Enforcement](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/2305-metrics-cardinality-enforcement) - v1.30
-    - [647 - APIServer Tracing](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/647-apiserver-tracing) - v1.30 -->
+- Stable
+  - [2305 - Dynamic Cardinality Enforcement](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/2305-metrics-cardinality-enforcement) - v1.30
 
 ## [Subprojects](https://git.k8s.io/community/sig-instrumentation#subprojects)
 
+**New in 2024:**
+
+- [zpages](https://git.k8s.io/kubernetes/staging/src/k8s.io/component-base/zpages)
 
 **Continuing:**
-  - custom-metrics-apiserver
-  - instrumentation
-  - instrumentation-addons
-  - instrumentation-tools
-  - klog
-  - kube-state-metrics
-  - metric-stability-framework
-  - metrics
-  - metrics-server
-  - prometheus-adapter
-  - structured-logging
-  - usage-metrics-collector
+
+- custom-metrics-apiserver
+- instrumentation
+- instrumentation-addons
+- instrumentation-tools
+- klog
+- kube-state-metrics
+- metric-stability-framework
+- metrics
+- metrics-server
+- prometheus-adapter
+- structured-logging
+- usage-metrics-collector
 
 ## [Working groups](https://git.k8s.io/community/sig-instrumentation#working-groups)
 
-**New in 2024:**
- - Serving
 **Continuing:**
- - Structured Logging
+
+- Serving
+- Structured Logging
 
 ## Operational
 
 Operational tasks in [sig-governance.md]:
-- [ ] [README.md] reviewed for accuracy and updated if needed
-- [ ] [CONTRIBUTING.md] reviewed for accuracy and updated if needed
-- [ ] Other contributing docs (e.g. in devel dir or contributor guide) reviewed for accuracy and updated if needed
-- [ ] Subprojects list and linked OWNERS files in [sigs.yaml] reviewed for accuracy and updated if needed
-- [ ] SIG leaders (chairs, tech leads, and subproject leads) in [sigs.yaml] are accurate and active, and updated if needed
-- [ ] Meeting notes and recordings for 2024 are linked from [README.md] and updated/uploaded if needed
 
+- [x] [README.md] reviewed for accuracy and updated if needed
+- [x] Other contributing docs (e.g. in devel dir or contributor guide) reviewed for accuracy and updated if needed
+- [x] Subprojects list and linked OWNERS files in [sigs.yaml] reviewed for accuracy and updated if needed
+- [x] SIG leaders (chairs, tech leads, and subproject leads) in [sigs.yaml] are accurate and active, and updated if needed
+- [ ] Meeting notes and recordings for 2024 are linked from [README.md] and updated/uploaded if needed
 
-[CONTRIBUTING.md]: https://git.k8s.io/community/sig-instrumentation/CONTRIBUTING.md
 [sig-governance.md]: https://git.k8s.io/community/committee-steering/governance/sig-governance.md
 [README.md]: https://git.k8s.io/community/sig-instrumentation/README.md
 [sigs.yaml]: https://git.k8s.io/community/sigs.yaml
-[devel]: https://git.k8s.io/community/contributors/devel/README.md
```

Comment:
It's probably okay to keep this as is?

Reply:
Ah it got auto-removed by markdownlint extension in vs code because this reference wasnt used anywhere, made a change now to actually reference this link, so its back now :)